ci-operator/step-registry/openstack/deprovision/credentials/openstack-deprovision-credentials-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+info() {
+	printf '%s: %s\n' "$(date --utc +%Y-%m-%dT%H:%M:%SZ)" "$*"
+}
+
+CLOUDS_YAML="${SHARED_DIR}/clouds.yaml"
+CLOUDS_YAML_UNRESTRICTED="${SHARED_DIR}/clouds-unrestricted.yaml"
+
+if ! [[ -f "$UNRESTRICTED_CLOUDS_YAML" ]]; then
```

Comment:
`UNRESTRICTED_CLOUDS_YAML` vs `CLOUDS_YAML_UNRESTRICTED` in line 12?

Reply:
woops! good catch, thanks.